### DISCUSSION TITLE
please god why are you doing this

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32245] [Other Feature Feedback, Questions, & Ideas] [answered]
I am trying to use git. why is there a social media feed and achievements?

**Comment by dermoumi** [comment, marked as answer]
There has been a positive feedback from young newcomers, so those are here to stay.

You have a few options:
- Add the html containers to your browser's adblock extension to hide them.
- Set your profile to private so that other people don't see yours.
- Just ignore them. You'll get to it eventually.

**Reply by 3kh0** [reply]
@thall-gnu It will help others if you mark this answer as the answer
